docs/guides/anomaly-detection-configuration/fail_on_zero.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "fail_on_zero"
+sidebarTitle: "fail_on_zero"
+---
+
+`fail_on_zero: true/false`
+
+Elementary anomaly detection tests will fail whether there is any zero metric value within the detection period.
+If undefined, default is false.
+
```

Comment:
maybe we can add a graph? Do we have something on Figma?

Reply:
[https://github.com/elementary-data/elementary/pull/1169#discussion_r1336915277](Same)